mysql-test/suite/galera/t/galera_garbd_invalid_donor_name.test

```diff
@@ -0,0 +1,94 @@
+# ==== Purpose ====
+#
+# This test verifies that garbd shuts down when when donor node is killed while
```

Comment:
Is this description OK?

Reply:
Thank you for pointing it out. Fixed in the next revision.